docs/testing/manual_testing/Crash Reporting.md

```diff
@@ -0,0 +1,14 @@
+The application will log crashes to external storage and can also include some information about these when a user contacts us for support.
+
+# Crashes Reported to External Storage
+1. Install the app
```

Comment:
Lol this is wrong.  I got interrupted while writing this and never came back to it.

Reply:
Yea, I know this too :)